hw/ip/otbn/dv/otbnsim/sim/loop.py

```diff
@@ -47,6 +47,18 @@ def __init__(self, start_addr: int, insn_count: int, restarts: int):
         self.start_addr = start_addr
         self.match_addr = start_addr + 4 * insn_count
 
+    @property
```

Comment:
Is there a strong reason to use properties rather than e.g. `get_loop_insn_addr()`? We've not used them elsewhere and I think I probably prefer being really explicit about what's the state that we're storing and what's derived from it.

Reply:
I like properties as they make refactorings such as the ones I've been doing here easier (no need to adjust the callers). But I really don't care much, switched over to getters/setters.